apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -138,10 +136,9 @@ export default function Chat({
       })
 
       return response
-    } catch (_) {
-      // do nothing
-    } finally {
+    } catch (error) {
       setResponseStream(undefined)
```

Comment:
You are not setting the stream to undefined if things go well now. Which was the previous behavior.

Reply:
Yes, it is inside the `try`, above the last `addMessage`. It was being called twice before.

Reply:
ah lol put it in finally that is the point